environs/tools/build.go

```diff
@@ -217,6 +217,9 @@ func copyExistingJujus(dir string) error {
 			return errors.Trace(err)
 		}
 	}
+	if skipCopyVersionFile {
```

Comment:
I can't see a test for this?

Reply:
right, added one more test to cover this